docs/source/asr/speaker_diarization/datasets.rst

```diff
@@ -1,72 +1,134 @@
+
+AMi Meeting Corpus
+~~~~~~~~~~~~~~~~~~
+
+The following are the suggested parameters for reproducing the diarization performance for AMI test set.
+
+.. code-block:: bash
+
+  diarizer.manifest_filepath="/path/to/AMItest_input_manifest.json"
+  diarizer.oracle_num_speakers=null # Performing unknown speaker case
+  diarizer.oracle_vad=True # Use oracle VAD extracted from RTTM files.
+  diarizer.collar=0.25
+  diarizer.ignore_overlap=True 
+  diarizer.speaker_embeddings.model_path = ``titanet_large`` 
+  diarizer.speaker_embeddings.window_length_in_sec=[3,1.5,1.0,0.5] # Multiscale setting
+  diarizer.speaker_embeddings.shift_length_in_sec=[1.5,0.75,0.5,0.25] # Multiscale setting 
+  diarizer.speaker_embeddings.parameters.multiscale_weights=[0.4,0.3,0.2,0.1] # More weights on the longer scales
+
+This setup is expected to reproduce speaker error rate  of 1.19% on AMI test set.
+
+To evaluate the performance on AMI Meeting Corpus, the following instructions can help.
+  - Download AMI Meeting Corpus from `AMI website <https://groups.inf.ed.ac.uk/ami/corpus/>`_.
+  - Get the test set (whitelist) from `Pyannotate AMI testset whitelist <https://raw.githubusercontent.com/pyannote/pyannote-audio/master/tutorials/data_preparation/AMI/MixHeadset.test.lst>`_.
+  - The merged RTTM file for AMI testset can be downloaded from `Pyannotate AMI testset RTTM file <https://raw.githubusercontent.com/pyannote/pyannote-audio/master/tutorials/data_preparation/AMI/MixHeadset.test.rttm>`_. Note that this file should be split into individual rttm files. Download split rttm files for AMI testset from `AMI testset split RTTM files <https://raw.githubusercontent.com/tango4j/diarization_annotation/main/AMI_corpus/test/split_rttms.tar.gz>`_.
+  - Generate an input manifest file using ``<NeMo_git_root>/scripts/speaker_tasks/pathsfiles_to_manifest.py``
+
+
+CallHome American English Speech (CHAES), LDC97S42: 2-speaker subset (CH109)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CH109 is a subset of the CHAES dataset which has only two speakers in one session. 
+The following are the suggested parameters for reproducing the diarization performance for the CH109 set.
+
+.. code-block:: bash
+
+  diarizer.manifest_filepath="/path/to/ch109_input_manifest.json"
+  diarizer.oracle_num_speakers=2 (Since there are exactly 2 speakers per each CH109 session)
+  diarizer.oracle_vad=True # Use oracle VAD extracted from RTTM files.
+  diarizer.collar=0.25
+  diarizer.ignore_overlap=True 
+  diarizer.speaker_embeddings.model_path = ``titanet_large`` 
+  diarizer.speaker_embeddings.window_length_in_sec=[1.5,1.0,0.5] # Multiscale setting
+  diarizer.speaker_embeddings.shift_length_in_sec=[0.75,0.5,0.25] # Multiscale setting
+  diarizer.speaker_embeddings.parameters.multiscale_weights=[0.33,0.33,0.33] # Equal weights
+
+This setup is expected to reproduce confusion error of 0.82% on CH109 set.
```

Comment:
I think we got best 0.68% right?

Reply:
No that's with titanet-m. Since you posted titanet-Large model everywhere, we should go with this. Actually, this is 0.94% in fact. I made a mistake with test datasets (tested with only subset of ch109)

Reply:
Perfect